crates/sui-core/src/traffic_controller/mod.rs

```diff
@@ -2,14 +2,19 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+pub mod nodefw_client;
+#[cfg(debug_assertions)]
+pub mod nodefw_test_server;
```

Comment:
Why is this conditionally configed? And if this is only for tests then likely we should have this as #[cfg(test)]

Reply:
Yeah should be `#[cfg(test)]`. Will fix